organizing_hub/models.py

```diff
@@ -37,6 +37,12 @@ def serve(self, request, *args, **kwargs):
             **kwargs
         )
 
+@unique
+class AlertLevels(Enum):
```

Comment:
This is much better 😂 If possible I suggest to use integer ids though, to decouple css class from db id.

Also this model is really for all bootstrap alerts, not just organizing hub, so I'm not sure exactly which app it belongs in?

Reply:
I was going to put it in pages but not sure if that makes sense either. So had to pick one.

Reply:
I thought about pages app too, but that's just for wagtail stuff, and this could be used basically anywhere we have a bootstrap alert, so basically anywhere on the site. It's almost like it should be part of a bootstrap app, or a front end assets app.

Reply:
interesting: https://django-bootstrap3.readthedocs.io/en/latest/templatetags.html#bootstrap-alert

Reply:
yup here it is: https://github.com/dyve/django-bootstrap3/blob/8.2.3/bootstrap3/templatetags/bootstrap3.py#L26
```
MESSAGE_LEVEL_CLASSES = {
    DEFAULT_MESSAGE_LEVELS.DEBUG: "alert alert-warning",
    DEFAULT_MESSAGE_LEVELS.INFO: "alert alert-info",
    DEFAULT_MESSAGE_LEVELS.SUCCESS: "alert alert-success",
    DEFAULT_MESSAGE_LEVELS.WARNING: "alert alert-warning",
    DEFAULT_MESSAGE_LEVELS.ERROR: "alert alert-danger",
}
```

Reply:
https://github.com/dyve/django-bootstrap3/blob/8.2.3/bootstrap3/components.py#L27

Reply:
Hmm, could reimplement the alerts to use render alert here but still need the enum to get choices from for CMS - importing those would be pretty brittle I think.

Reply:
I think for now it makes most sense in pages, creating a new front-end app is way outside scope here.

Reply:
I'm not sure how this is making things simpler over just using a config similar to how django bootstrap implemented? What is enum trying to solve here? Seems to be making things much more complex.

Reply:
I'm not asking you to refactor this right now to use the django bootstrap library. I'm just pointing out that it is one way to do it. The most important thing at the moment I think is that this should be an Enum and not a setting.

Personally I think putting all our Enums into a single settings file is the more complex option. Settings is not a catchall for any constant/variable we have in the app which is not stored in the db. It should be used for things which actually fit the idea of a config setting, so we can have proper separation of layers.

Django bootstrap is not using the settings file, it is using template tags. It has a slightly different use case and they decided to use a dictionary instead of Enum, I don't know why.